aviary/subsystems/aerodynamics/flops_based/test/test_mux_component.py

```diff
@@ -58,7 +58,7 @@ def test_mux(self):
         prob.set_val(Aircraft.Nacelle.LAMINAR_FLOW_LOWER, np.array([0.10]))
 
         prob.run_model()
-        prob.check_partials(compact_print=True, method="cs")
+        # prob.check_partials(compact_print=True, method="cs")
```

Comment:
Is this purposefully commented out?

Reply:
I should have kept it. It is uncommented out now.